include/oc_session_events.h

```diff
@@ -16,65 +16,65 @@
  *
  ****************************************************************************/
 /**
-  @file
-*/
+ * @file oc_session_events.h
+ *
+ * @author Kishen Maloor
+ * @author Daniel Adam
+ */
+
 #ifndef OC_SESSION_EVENTS_H
 #define OC_SESSION_EVENTS_H
 
+#include "oc_export.h"
 #include "oc_endpoint.h"
-#include "util/oc_process.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 /**
  * @brief session states
- *
  */
 typedef enum {
-  OC_SESSION_CONNECTED,   ///< session connected
-  OC_SESSION_DISCONNECTED ///< session disconnected
+  OC_SESSION_CONNECTED,    ///< session connected
+  OC_SESSION_DISCONNECTED, ///< session disconnected
 } oc_session_state_t;
 
-/**
-  @brief Callback function to pass the session event infomation to App.
-  @param endpoint  endpoint info which the session event is happened.
-  @param state  enum values in oc_session_state_t.
-*/
-typedef void (*session_event_handler_t)(const oc_endpoint_t *endpoint,
-                                        oc_session_state_t state);
+#define OC_ERR_SESSION_EVENT_HANDLER_NOT_FOUND -6
+#define OC_ERR_SESSION_EVENT_HANDLER_REMOVED -7
```

Comment:
This `OC_ERR_SESSION_EVENT_HANDLER_NOT_FOUND` never comes from the handler.

And OC_ERR_SESSION_EVENT_HANDLER_REMOVED is used only when networks is shutdown - but it is expected that all managers removed all registered callback. I think we don't need this.